suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -0,0 +1,58 @@
+import { describe, test, beforeAll, expect } from '@jest/globals'
+import { newCeramic } from '../../utils/ceramicHelpers.js'
+import { createDid } from '../../utils/didHelper.js'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { Model } from '@ceramicnetwork/stream-model'
+import { ModelInstanceDocument } from '@ceramicnetwork/stream-model-instance'
+import { newModel, basicModelDocumentContent } from '../../models/modelConstants'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { CommonTestUtils as TestUtils } from '@ceramicnetwork/common-test-utils'
+import { utilities } from '../../utils/common.js'
+
+const delay = utilities.delay
+const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
+// TODO : Update to the modelId created manually on all envs, if id cannot be the same then create an array for different envs
+const adminSeeds = String(process.env.COMPOSEDB_ADMIN_DID_SEEDS).split(',')
+const nodeSyncWaitTimeSec = 2
+
+describe('Model Integration Test', () => {
+  let ceramicNode1: CeramicClient
+  let ceramicNode2: CeramicClient
+  let modelId: StreamID
+  beforeAll(async () => {
+    const did1 = await createDid(adminSeeds[0])
+    let did2
+    if (adminSeeds[1]) {
```

Comment:
can condense the below  if else to ` const did2 = adminSeeds[1] ? await createDid(adminSeeds[1]) : did1`